CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# X.X.X - [XXXX-XX-XX]
+
+### `Added`
+
+### `Fixed`
+
+- Input to BootsrapAnn is now supplied in a single channel [#151](https://github.com/genomic-medicine-sweden/tomte/pull/151)
```

Comment:
If this caused sample mixups in previous versions of the pipeline (?), could you add this to the text?

Reply:
Good point!